rest-server/test/jobExecutionType.js

```diff
@@ -102,17 +102,17 @@ describe('Job execution type API /api/v1/jobs/:jobName/executionType', () => {
       });
   });
 
-  it('should not stop other user\'s job', (done) => {
+  it('admin should stop other user\'s job successfully', (done) => {
     chai.request(server)
```

Comment:
Suggest you do not change the normal user unauthorized test case but keep it and add a new admin test case.